scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3710,7 +3713,8 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     } else if (token.is[LeftBrace]) {
       inBraces(templateStats())
     } else {
-      newLineOpt()
+      while (token.is[LF] || token.is[LFLF])
```

Comment:
tokenizer should never produce 2 consecutive LF / LFLF combinations:
2x LF => LFLF
2x LFLF => LFLF

`while` should not be needed, `newLinesOpt()` should be enough

Reply:
Changed and added more newlines, thanks!